apps/coscrad-frontend/src/components/COSCRADLogo/COSCRADLogo.tsx

```diff
@@ -0,0 +1,10 @@
+import coscradLogo from '../../assets/coscrad-logo.png';
```

Comment:
Is this really how its done in React? This looks extremely odd. Let's talk about this at standup.

Reply:
I found that on a blog, there are a variety of ways to do this.  The reason I went for this version is that the import will break the build if the image isn't present so it would prevent a broken image link.